### PR TITLE
[Feature]: add TensorRT InstanceNormalization plugin

### PR DESCRIPTION
**Pull request by RunningLeon**
This PR includes:

- [x] Add `InstanceNormalization` plugin that supports dynamic shape in TensorRT.
- [x] Update relevant docs.

### CODE REVIEW

File: tests/test_ops/test_tensorrt.py [hunk 1]
```diff
@@ -478,3 +478,88 @@ def func(input, grid):
     if os.path.exists(trt_file):
         os.remove(trt_file)
     assert torch.allclose(pytorch_results, trt_results)
+
+
+@pytest.mark.parametrize('dynamic_export', [True, False])
+def test_instance_norm(dynamic_export):
+
+    n, c, h, w = 2, 3, 10, 10
+    norm = nn.InstanceNorm2d(c, affine=True).cuda()
```
Comment (grimoire) [inline tests/test_ops/test_tensorrt.py:487]: add `.eval()` here and remove requires_grad below.

File: tests/test_ops/test_tensorrt.py [hunk 2]
```diff
@@ -478,3 +478,88 @@ def func(input, grid):
...
+             list(data.shape), [2 * n, c, 2 * h, 2 * w]],
+        }
+    else:
+        opt_shape_dict = {
+            'input': [list(data.shape),
+                      list(data.shape),
+                      list(data.shape)],
+        }
+    # trt config
+    fp16_mode = False
```
Comment (grimoire) [inline tests/test_ops/test_tensorrt.py:540]: Since this plugin support fp16, should we add fp16 test?

**Review by grimoire — APPROVED** [review]
LGTM!

File: tests/test_ops/test_tensorrt.py [hunk 3]
```diff
@@ -575,3 +575,83 @@ def test_cummin_cummax(func: Callable):
...
+    max_workspace_size = 1 << 30
+
+    trt_engine = onnx2trt(
+        onnx_model,
+        opt_shape_dict,
+        fp16_mode=fp16_mode,
+        max_workspace_size=max_workspace_size)
+
+    save_trt_engine(trt_engine, trt_file)
+    trt_model = TRTWraper(trt_file, input_names, output_names)
```
Comment (ZwwWayne) [inline tests/test_ops/test_tensorrt.py:642]: Use TRTWrapper

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**